include/dynamic-graph/logger.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2015, 2019
+ * LAAS-CNRS
+ * Andrea Del Prete, François Bailly,
+ *
+ * This file is part of dynamic-graph.
+ * See license file.
+ */
+
+#ifndef __dynamic_graph_logger_H__
+#define __dynamic_graph_logger_H__
+
+/* --------------------------------------------------------------------- */
+/* --- API ------------------------------------------------------------- */
+/* --------------------------------------------------------------------- */
+
+#if defined (WIN32)
+#  if defined (logger_EXPORTS)
+#    define LOGGER_EXPORT __declspec(dllexport)
+#  else
+#    define LOGGER_EXPORT __declspec(dllimport)
+#  endif
+#else
+#  define LOGGER_EXPORT
+#endif
```

Comment:
Can you remove those define ? If you need them, they are already in `dynamic-graph/config.hh`.